addons/addon-raas-s3-copy/packages/s3-synchronizer/README.md

```diff
@@ -24,8 +32,12 @@ Usage of bin/s3-synchronizer-darwin-amd64:
         Whether to print debug information
   -destination string
         The directory to download to (default "./")
-  -pollInterval int
-        The delay (in seconds) between api http requests (default 60)
+  -recurringDownloads 
+        Whether to periodically download changes from S3 (default false)
+  -stopRecurringDownloadsAfter int
+        Stop recurring downloads after certain number of seconds. ZERO or Negative value means continue indefinitely. (default -1 i.e., indefinitely)
```

Comment:
What is the motivation behind having this flag for customers. I understand the need for this for say unit tests, but are there cases when customers would want to stop downloads if recurringDownloads is set to true?

Reply:
Customer may have some temporarily running job pushing changes to S3 and keep monitoring S3 beyond that duration may be waste of CPU/Memory on the workspace instance if no more changes are expected to happen in S3. Having such a flag would allow for auto-stopping of the monitoring thread. 

Another use case may be some studies (say open data studies) which are not expected to change at all. There is no need to keep monitoring that bucket on running workspace. Having this parameter allows for stopping the monitoring thread pretty much immediately after the initial download in those cases.

Reply:
I see, let's keep it in that case. If you think these details will be user, feel free to add them to documentation as well.